nomenclature/definitions/region/subcountries.yaml

```diff
@@ -302,28 +302,58 @@ United Kingdom|North Sea:
     ehighway2050: 106_NS+107_NS+108_NS+109_NS
     plan4res: NSUK
     
-# List of electricity price regions in Norway
-Norway|Oslo:
-   definition: this is the electricity price region 1 in Norway 
+# List of subregions in Norway, designed to best decribe the hydropower system in Norway
```

Comment:
Not sure if mentioning hydropower instead of electricity is helpful...

Reply:
Ok, good point. I was explained that the regions are defined in a way to best respect the connections between different hydro power plants along a watercourse, hence not necessarily coinciding with electricity price regions are nuts regions. Maybe we could write "List of Norwegian sub-regions, designed to enable detailed modelling of the hydro power production system"? Do you think that would be better?

Reply:
Maybe the following...
```suggestion
# List of subregions in Norway
# based on the regional electricity grid and key watersheds for hydropower generation
```

Reply:
I adjusted it